src/components/Header/Header.tsx

```diff
@@ -0,0 +1,57 @@
+import { IcBack, IcCart, IcLogo, IcMenu } from "@assets/svg";
+import {
+  headerCenterSectionWrapper,
+  headerLayout,
+  headerLeftIconStyle,
+  headerLeftSectionWrapper,
+  headerLogoStyle,
+  headerRightIconStyle,
+  headerRightSectionWrapper,
+} from "@components/Header/Header.style";
+import routePath from "@routes/routePath";
+import { useNavigate } from "react-router-dom";
+
+interface HeaderProps {
+  isPrevButton?: boolean;
+}
+
+const Header = ({ isPrevButton = false }: HeaderProps) => {
+  const navigate = useNavigate();
+
+  const handleLogoClick = () => {
+    navigate(routePath.HOME);
+  };
+
+  const handleBackClick = () => {
+    if (window.history.length > 2) {
+      navigate(-1);
+    } else {
+      navigate(routePath.HOME);
+    }
+  };
+
+  return (
+    <>
+      <div css={headerLayout}>
+        {isPrevButton ? (
+          <section css={headerLeftSectionWrapper}>
+            <IcBack css={headerLeftIconStyle} onClick={handleBackClick} />
+          </section>
+        ) : (
+          <div></div>
+        )}
+
+        <section css={headerCenterSectionWrapper}>
+          <IcLogo css={headerLogoStyle} onClick={handleLogoClick} />
+        </section>
+
+        <section css={headerRightSectionWrapper}>
+          <IcCart css={headerRightIconStyle} />
+          <IcMenu css={headerRightIconStyle} />
+        </section>
+      </div>
+    </>
```

Comment:
불필요한 빈 태그는 제거하고, header 태그를 이용하면 헤더라는 의미를 더욱 잘 나타낼 수 있어보여요 !
```suggestion
      <header css={headerLayout}>
        {isPrevButton ? (
          <section css={headerLeftSectionWrapper}>
            <IcBack css={headerLeftIconStyle} onClick={handleBackClick} />
          </section>
        ) : (
          <div></div>
        )}

        <section css={headerCenterSectionWrapper}>
          <IcLogo css={headerLogoStyle} onClick={handleLogoClick} />
        </section>

        <section css={headerRightSectionWrapper}>
          <IcCart css={headerRightIconStyle} />
          <IcMenu css={headerRightIconStyle} />
        </section>
      </header>
```

Reply:
오오 너무 좋네요 !! 불필요한 태그의 경우 grid 영역 잡으려고 놔뒀는데, 민하님 리뷰 보고 다시 생각해보니 section 안에서 조건부 렌더링을 하면, 굳이 빈 태그를 놔두지 않아도 영역이 잡히네요! 😊 반영 완료입니다 -! 꼼꼼한 리뷰 감사해요!